CHANGELOG.md

```diff
@@ -2,8 +2,21 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
-## Beta
-[This release is in beta. Click here to see changes since 4.1.3.](https://github.com/awslabs/service-workbench-on-aws/compare/v4.1.3...mainline)
+
+### [**4.2.0**](https://github.com/awslabs/service-workbench-on-aws/compare/v4.1.3...v4.2.0) (2022-01-19)
+Customer Impact:
+* As a security enhancement, the `internal` authentication method used by Service Workbench (the legacy default authentication method) will soon be deprecated. 
+* After installing this release, the native Amazon Cognito user pool will soon become the default authentication method, and will be reflected accordingly on the application's login page (alongside your external SAML IdP integrations, if any).
+* You will find the default (but user-customizable) configurations determining the native Amazon Cognito user pool behavior in the `main/solution/post-deployment/config/settings/.defaults.yml` file. 
+* By default, users can sign up themselves onto the native pool, but can only access Service Workbench once they are approved by the application admin. The user addition experience on Service Workbench for native Amazon Cognito user pool is similar to that of an external IdP.
```

Comment:
If using native Amazon Cognito user pool, users can sign up for a user account, but can not  access Service Workbench until they are approved by the application admin.